joomla/joomla-cms
# [5.2] Allow nested Subform inside a Subform Custom Field

### Summary of Changes

When creating a custom field of type Subform, you can select the child fields that you want as part of the Subform.

In the current situation, the dropdown of fields selection doesn't allow to select a Subform to be nested inside a Subform. This was probably done to avoid cluttering the interface too much if somebody wanted to do it.

But there is a legimitate use for this and somebody may want to intentionally have a nested Subform inside a Subform. I am currently having a legitimate use case where I want a Subform inside a Subform and I can't do it because of this limitation. There is no reason to have this limitation, let the user do it if they want to.

### Testing Instructions

- Go to Content -> Fields
- Make sure you already have a bunch of created fields, as well as a Subform field already created
- Create a new field of type Subform
- Look at the "Fields" area to see what child fields you can include with the dropdown

### Actual result BEFORE applying this Pull Request

- A field of type subform is not in the available list of fields in the dropdown

### Expected result AFTER applying this Pull Request

- A field of type subform now is available to be selected

### Link to documentations

- No documentation changes for docs.joomla.org needed
